TaskHouseApi/Migrations/20181021121258_InitialCreate.cs

```diff
@@ -4,45 +4,45 @@
 
 namespace TaskHouseApi.Migrations
 {
-    public partial class Initial_structure : Migration
+    public partial class InitialCreate : Migration
```

Comment:
Just for future reference, migrations should never change after creation. If something in the DB needs to be updated a new migration is required. This is especially important when the system is running in production. In dev we can just delete the db and update it with the new fields, but it is good practice to create a new migration to mange db updates.